src/components/breadcrumb/breadcrumb-step/breadcrumb-step.js

```diff
@@ -65,34 +54,52 @@ const returnLabel = (current, id) => {
   }
   return styles.label;
 };
-export const breadcrumb = props => (
-  <Link to={props.link} className={styles.link}>
-    <Grid container className={styles.column}>
-      <MyButton className={returnIcon(props.current, props.selection, props.value)}>
-        <SvgIcon className={returnSvg(props.current, props.selection, props.value)}>
-          <path d={returnStatus(props.selection, props.value)} />
-        </SvgIcon>
-      </MyButton>
-      {(() => {
-        switch (props.value) {
-          case 0:
-            return <div className={returnRightBar(props.selection, props.value)} />;
-          case 4:
-            return <div className={returnLeftBar(props.selection, props.value - 1)} />;
-          default:
-            return (
-              <div>
-                <div className={returnLeftBar(props.selection, props.value - 1)} />
-                <div className={returnRightBar(props.selection, props.value)} />
-              </div>
-            );
-        }
-      })()}
+export const breadcrumb = (props) => {
+  const returnStatus = (selection, id) => {
+    if (selection > id) {
+      return <CheckmarkIcon className={returnSvg(props.current, props.selection, props.value)} />;
+    } else if (id === 0) {
+      return (
+        <QuestionMarkIcon className={returnSvg(props.current, props.selection, props.value)} />
+      );
+    } else if (id === 1) {
+      return <NotesIcon className={returnSvg(props.current, props.selection, props.value)} />;
+    } else if (id === 2) {
+      return <LocationIcon className={returnSvg(props.current, props.selection, props.value)} />;
+    } else if (id === 3) {
+      return <DollarSignIcon className={returnSvg(props.current, props.selection, props.value)} />;
+    } else if (id === 4) {
+      return <Notes2Icon className={returnSvg(props.current, props.selection, props.value)} />;
+    }
+    return <QuestionMarkIcon className={returnSvg(props.current, props.selection, props.value)} />;
```

Comment:
This should be a `switch`

Reply:
Ya we need to refactor that logic in a new branch. It was done a while ago

Reply:
The original diff showed this was a `switch`. It's not a hard change to make (no tests should change), so I'd like to see that done in this pr.

Reply:
done